packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as path from 'path';
+import { projectPaths, workspaceUtils } from '../../../src/';
+
+jest.mock('@salesforce/core', () => {
+  return {
+    Global: {
+      SFDX_STATE_FOLDER: '.sfdx'
+    }
+  };
+});
+
+describe('test project paths', () => {
+  const FAKE_WORKSPACE = '/here/is/a/fake/path/to/';
+
+  describe('test stateFolder', () => {
+    const FAKE_STATE_FOLDER = '.sfdx';
+    let getRootWorkspacePathStub: jest.SpyInstance;
+    let hasRootWorkspaceStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      getRootWorkspacePathStub = jest.spyOn(workspaceUtils, 'getRootWorkspacePath').mockReturnValue(FAKE_WORKSPACE);
+    });
+
+    it('is defined', () => {
```

Comment:
nit: curious on your thoughts.  We've have somewhat of a standard of having all unit tests follow the `should do a thing` pattern where each it starts with `should`.  Do you think we follow that pattern here?

Reply:
oh! Yes I can change that. This pattern felt more intuitive/declarative, but I'm not opposed to the extra chars of the "should"